From 36c043f65b3a039c51f7f4bb4607d56fcc1331ba Mon Sep 17 00:00:00 2001
From: Joshua Strobl <joshua@stroblindustries.com>
Date: Sat, 17 Feb 2018 14:12:21 +0200
Subject: [PATCH 1/1] Remove author revision prompt since our use of Arcanist
 is primarily for patch submission and landing. These patch landings are
 typically done by other indications which are not the creators of said
 packages, thus an additional unnecessary prompt is well, unnecessary.

---
 src/workflow/ArcanistCommitWorkflow.php |  7 -------
 src/workflow/ArcanistLandWorkflow.php   | 19 -------------------
 2 files changed, 26 deletions(-)

diff --git a/src/workflow/ArcanistCommitWorkflow.php b/src/workflow/ArcanistCommitWorkflow.php
index c0e91db3..f50b9a73 100644
--- a/src/workflow/ArcanistCommitWorkflow.php
+++ b/src/workflow/ArcanistCommitWorkflow.php
@@ -323,13 +323,6 @@ EOTEXT
         $revision_title);
     }
 
-    if ($revision['authorPHID'] != $this->getUserPHID()) {
-      $confirm[] = pht(
-        "You are not the author of '%s: %s'. Commit this revision anyway?",
-        "D{$revision_id}",
-        $revision_title);
-    }
-
     $revision_source = idx($revision, 'branch');
     $current_source = $repository_api->getBranchName();
     if ($revision_source != $current_source) {
-- 
2.16.1

